pkg/sql/opt/memo/testdata/stats/scan

```diff
@@ -1091,16 +1091,16 @@ project
  ├── fd: ()-->(6)
  ├── index-join t47742
  │    ├── columns: a:1(int) t47742.b:2(bool!null)
- │    ├── stats: [rows=3063.5, distinct(2)=1.0025, null(2)=0]
- │    │   histogram(2)=  0    0    0.0024693 3063.5
- │    │                <--- false ----------- true
+ │    ├── stats: [rows=3063.5, distinct(2)=1, null(2)=0]
+ │    │   histogram(2)=  0 3063.5
+ │    │                <--- true
  │    ├── fd: ()-->(2)
  │    └── scan t47742@b_idx
  │         ├── columns: t47742.b:2(bool!null) rowid:3(int!null)
  │         ├── constraint: /-2/3: [/true - /true]
- │         ├── stats: [rows=3063.5, distinct(2)=11, null(2)=0]
- │         │   histogram(2)=  0   0    0.0024693 3063.5
- │         │                <--- NULL ----------- true
+ │         ├── stats: [rows=3063.5, distinct(2)=1, null(2)=0]
```

Comment:
Not sure exactly why this changed, but it looks like the distinct count is correctly `1` now instead of `11`.